src/report.ts

```diff
@@ -1,11 +1,11 @@
 import fs from 'fs';
-import type { DiffResult } from 'compare';
+import { VRTResultsEntries } from 'index';
```

Comment:
this is circular dependency. You can export VRTResultsEntries to separate file and import it, where it is needed.